pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, protobuf
+, bzip2
+, openssl
+, sqlite
+, zstd
+, stdenv
+, darwin
+, nix-update-script
+, nixosTests
+, rocksdb_8_11
+}:
+
+let
```

Comment:
That file looks for me quite the same like the stalwart-mail_0_6. Correct me if i am wrong here.
But cant we just override the versions, so that we dont have to keep 2 times mostly identical build instructions?

So we could keep the "newer" stalwart package in the package thing and override to the 0_6 in the top-level/all-pckage, cant we?

Reply:
good point, fixed it :)